### PR TITLE
Created project for restaurant management application

### PR DESCRIPTION
**Pull request by shubhaangi04**
Added the project structure and boilerplate code

### CODE REVIEW

File: restaurant-management-system/restaurant-management-app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,45 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '3.3.2'
+	id 'io.spring.dependency-management' version '1.1.6'
+}
+
+group = 'com.vikas-open-source'
```
Comment (vikas-maersk) [inline restaurant-management-system/restaurant-management-app/build.gradle:7]: let's use com.thisissvikas

File: restaurant-management-system/restaurant-management-app/.gitignore [hunk 2]
```diff
@@ -0,0 +1,37 @@
+HELP.md
```
Comment (vikas-maersk) [inline restaurant-management-system/restaurant-management-app/.gitignore:1]: Do we need the .DS_Store file?

**Review by vikas-maersk — CHANGES_REQUESTED** [review]
Can you create a simple SpringBootTest class that just tests the loading of context? That will ensure that we can build the app and see if its starting up at build time?

File: restaurant-management-system/restaurant-management-app/src/main/java/com/thisissvikas/restaurant_management_app/controller/HelloWorldController.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloWorldController {
+
+    @GetMapping("/health")
+    public ResponseEntity<String> hello() {
```
Comment (thisissvikas) [inline restaurant-management-system/restaurant-management-app/src/main/java/com/thisissvikas/restaurant_management_app/controller/HelloWorldController.java:11]: let's call this class HealthCheckController and accordingly name the method

File: restaurant-management-system/restaurant-management-app/src/main/java/com/thisissvikas/restaurant_management_app/controller/HelloWorldController.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package com.thisissvikas.restaurant_management_app.controller;
```
Comment (thisissvikas) [inline restaurant-management-system/restaurant-management-app/src/main/java/com/thisissvikas/restaurant_management_app/controller/HelloWorldController.java:1]: Is this package naming valid?

**Review by thisissvikas — CHANGES_REQUESTED** [review]

File: restaurant-management-system/restaurant-management-app/src/main/java/com/thisissvikas/restaurant_management_app/controller/HelloWorldController.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloWorldController {
+
+    @GetMapping("/health")
```
Comment (thisissvikas) [inline restaurant-management-system/restaurant-management-app/src/main/java/com/thisissvikas/restaurant_management_app/controller/HelloWorldController.java:10]: let's maintain API versioning

**Review by thisissvikas — APPROVED** [review]

**Review by thisissvikas — APPROVED** [review]

**Pull request merged**